tests/forms_tests/tests/test_media.py

```diff
@@ -1,8 +1,90 @@
 from django.forms import CharField, Form, Media, MultiWidget, TextInput
+from django.forms.widgets import MediaAsset, Script
 from django.template import Context, Template
-from django.templatetags.static import static
 from django.test import SimpleTestCase, override_settings
-from django.utils.html import format_html, html_safe
+from django.utils.html import html_safe
+
+
+class CSS(MediaAsset):
+    element_template = '<link href="{path}"{attributes}>'
+
+    def __init__(self, href, **attributes):
+        super().__init__(href, **attributes)
+        self.attributes["rel"] = "stylesheet"
+
+
+@override_settings(
+    STATIC_URL="http://media.example.com/static/",
+)
+class MediaAssetTestCase(SimpleTestCase):
+    def test_init(self):
+        attributes = {"media": "all", "is": "magic-css"}
+        asset = MediaAsset("path/to/css", **attributes)
+        self.assertEqual(asset._path, "path/to/css")
+        self.assertEqual(asset.attributes, attributes)
+        self.assertIsNot(asset.attributes, attributes)
+
+    def test_eq(self):
+        self.assertEqual(MediaAsset("path/to/css"), MediaAsset("path/to/css"))
+        self.assertEqual(MediaAsset("path/to/css"), "path/to/css")
+        self.assertEqual(
+            MediaAsset("path/to/css", media="all"), MediaAsset("path/to/css")
+        )
+
+        self.assertNotEqual(MediaAsset("path/to/css"), MediaAsset("path/to/other.css"))
+        self.assertNotEqual(MediaAsset("path/to/css"), "path/to/other.css")
+        self.assertNotEqual(MediaAsset("path/to/css", media="all"), CSS("path/to/css"))
+
+    def test_hash(self):
+        self.assertEqual(hash(MediaAsset("path/to/css")), hash("path/to/css"))
+        self.assertEqual(
+            hash(MediaAsset("path/to/css")), hash(MediaAsset("path/to/css"))
+        )
+
+    def test_str(self):
+        self.assertEqual(
+            str(MediaAsset("path/to/css")),
+            "http://media.example.com/static/path/to/css",
+        )
+        self.assertEqual(
+            str(MediaAsset("http://media.other.com/path/to/css")),
+            "http://media.other.com/path/to/css",
+        )
+
+    def test_repr(self):
+        self.assertEqual(repr(MediaAsset("path/to/css")), "MediaAsset('path/to/css')")
+        self.assertEqual(
+            repr(MediaAsset("http://media.other.com/path/to/css")),
+            "MediaAsset('http://media.other.com/path/to/css')",
+        )
+
+    def test_path(self):
+        asset = MediaAsset("path/to/css")
+        self.assertEqual(asset.path, "http://media.example.com/static/path/to/css")
+
+        asset = MediaAsset("http://media.other.com/path/to/css")
+        self.assertEqual(asset.path, "http://media.other.com/path/to/css")
+
+
+@override_settings(
+    STATIC_URL="http://media.example.com/static/",
+)
+class ScriptTestCase(SimpleTestCase):
+
+    def test_init__with_src_kwarg(self):
```

Comment:
```suggestion
@override_settings(STATIC_URL="http://media.example.com/static/")
class ScriptTestCase(SimpleTestCase):
    def test_init_with_src_kwarg(self):
```
(nit/personal preference)